### PR TITLE
use pytest-cov

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

seems that the updated `cov` extension covers subprocesses

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by tchaton — APPROVED** [review]
LGTM !

File: requirements/test.txt [hunk 1]
```diff
@@ -1,7 +1,8 @@
-coverage>=5.0
+coverage>=5.2
```
Comment (carmocca) [inline requirements/test.txt:1]: do we still need this? isn't `pytest-cov` enough?

Comment (Borda) [reply requirements/test.txt:1]: well, it is installed under the hood anyway :]

**Review by carmocca — APPROVED** [review]

File: requirements/test.txt (hunk 1 above)
Comment (carmocca) [reply requirements/test.txt:1]: it would be nice to prune extra dependencies, even if they are installed under the hood

Comment (Borda) [reply requirements/test.txt:1]: I see your point, but that could be a bit tricky as we have many cached in CI so removing some user may found it is needed in the end...
btw, even in the pytest extension, they suggest to pin `coverage>=5.2`

**Review by SeanNaren — APPROVED** [review]

**Pull request merged**